tests/rptest/tests/topic_recovery_test.py

```diff
@@ -1154,7 +1154,7 @@ def verify():
 
         wait_until(verify,
                    timeout_sec=timeout.total_seconds(),
-                   backoff_sec=1,
+                   backoff_sec=5,
```

Comment:
The net effect of this change is that the `deltas` object is calculated over a 5 times longer period?

Reply:
time between checks now 5 time larger
It was 1 sec before